tests/softreboot_test.go

```diff
@@ -151,20 +149,21 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 
 		When("soft reboot vmi after paused and unpaused via virtctl", func() {
 			It("should failed to soft reboot a paused vmi", func() {
-				runVMI(true, true)
+				vmi = libvmi.NewTestToolingFedora()
+				vmi = tests.RunVMIAndExpectLaunch(vmi, 360)
 				tests.WaitAgentConnected(virtClient, vmi)
 
 				command := tests.NewRepeatableVirtctlCommand(virtctlpause.COMMAND_PAUSE, "vmi", "--namespace", util.NamespaceTestDefault, vmi.Name)
 				Expect(command()).To(Succeed())
-				tests.WaitForVMICondition(virtClient, vmi, v1.VirtualMachineInstancePaused, 30)
+				tests.WaitForVMICondition(virtClient, vmi, v1.VirtualMachineInstancePaused, 360)
```

Comment:
is this an attempt at making the timeouts uniform across the file ? 

Could you export the value to a const ? It's repeated all across the file - it could be reused.

Reply:
no, you spotted here a careless s/earch/replace/ bug of mine.